### PR TITLE
Defs/regions

### PR DESCRIPTION
**Pull request by sandrinecharousset**
Added additionnal regions for plan4eu:
Asia and Africa for dealing with imports
North sea clusters for offshore wind

### CODE REVIEW

**Comment by danielhuppmann** [comment]
Looks good @sandrinecharousset - and it seems the unit test was successful in identifying  parsing issues that would have caused a lot of headache later...

Please assign me as a reviewer (top-right of this page) when you are done and want me to review in detail!

**Comment by sandrinecharousset** [comment]
> Looks good @sandrinecharousset - and it seems the unit test was successful in identifying parsing issues that would have caused a lot of headache later...
> 
> Please assign me as a reviewer (top-right of this page) when you are done and want me to review in detail!

Haha, those tests are usefull but here it was not so easy to find where the bugs were....

File: nomenclature/definitions/region/subcountries.yaml [hunk 1]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
   note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; ITN+ITS=IT
   ehighway2050: 54_IT + 55_IT + 56_IT + 98_IT
   plan4res: ITS
+  
+# ehighway north sea clusters
+Belgium|North sea: 
+    definition: this 'virtual' region holds the offshore windpower of belgium in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions;
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:213]:
```suggestion
    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
```

File: nomenclature/definitions/region/subcountries.yaml [hunk 2]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+    definition: this 'virtual' region holds the offshore windpower of belgium in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; 
+    ehighway2050: 110_NS
+    plan4res: NSBE
+    
+Germany|North sea: 
+    definition: this 'virtual' region holds the offshore windpower of germany in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions;
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:220]:
```suggestion
    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
```

File: nomenclature/definitions/region/subcountries.yaml [hunk 3]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+    definition: this 'virtual' region holds the offshore windpower of germany in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; 
+    ehighway2050: 112_NS
+    plan4res: NSDE
+
+The Netherlands|North sea: 
+    definition: this 'virtual' region holds the offshore windpower of the netherlands in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions;
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:227]:
```suggestion
    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
```

File: nomenclature/definitions/region/subcountries.yaml [hunk 4]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+    definition: this 'virtual' region holds the offshore windpower of the netherlands in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; 
+    ehighway2050: 111_NS
+    plan4res: NSNL
+    
+United Kingdom|North sea: 
+    definition: this 'virtual' region holds the offshore windpower of the UK in norh sea 
+    country: Belgium
+    note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions;
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:234]:
```suggestion
    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
```

File: nomenclature/definitions/region/subcountries.yaml [hunk 5]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
   note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; ITN+ITS=IT
   ehighway2050: 54_IT + 55_IT + 56_IT + 98_IT
   plan4res: ITS
+  
+# ehighway north sea clusters
+Belgium|North sea:
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:210]: I would think that `North Sea` is more in line with our convention for spelling variables...?

Comment (sandrinecharousset) [reply nomenclature/definitions/region/subcountries.yaml:210]: @danielhuppmann agree :-)

**Comment by sandrinecharousset** [comment]
> thanks @sandrinecharousset - very helpful! A few minor comments an fixes for consistency, then good to be merged!

@danielhuppmann I have included your change requests. Thanks

File: nomenclature/definitions/region/subcountries.yaml [hunk 6]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+Belgium|North Sea: 
+    definition: this 'virtual' region holds the offshore windpower of belgium in norh sea 
+    country: Belgium
+    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
+    ehighway2050: 110_NS
+    plan4res: NSBE
+    
+Germany|North Sea: 
+    definition: this 'virtual' region holds the offshore windpower of germany in norh sea 
+    country: Belgium
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:219]:
```suggestion
    country: Germany
```

File: nomenclature/definitions/region/subcountries.yaml [hunk 7]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
+    ehighway2050: 112_NS
+    plan4res: NSDE
+
+The Netherlands|North Sea: 
+    definition: this 'virtual' region holds the offshore windpower of the netherlands in norh sea 
+    country: Belgium
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:226]: The Netherlands

File: nomenclature/definitions/region/subcountries.yaml [hunk 8]
```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
...
+    notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
+    ehighway2050: 111_NS
+    plan4res: NSNL
+    
+United Kingdom|North Sea: 
+    definition: this 'virtual' region holds the offshore windpower of the UK in norh sea 
+    country: Belgium
```
Comment (danielhuppmann) [inline nomenclature/definitions/region/subcountries.yaml:233]:
```suggestion
    country: United Kingdom
```

**Comment by sandrinecharousset** [comment]
> just noticed one other issue...

oups, that was stupid copy-paste..... 
this is done

**Pull request merged**